test/e2e/e2e.spec.ts

```diff
@@ -0,0 +1,67 @@
+import crypto from 'crypto';
+import { execSync, spawn } from 'child_process';
+import { AutoblocksAPIClient } from '../../src';
+
+jest.setTimeout(60_000);
+
+async function waitForTraceToExist(traceId: string): Promise<void> {
+  const apiClient = new AutoblocksAPIClient();
+
+  let numTries = 30;
+  while (numTries) {
+    try {
+      await apiClient.getTrace({ traceId });
+      console.log(`Found trace ${traceId}`);
+      break;
+    } catch (err) {
+      console.log(`Didn't find trace ${traceId}, ${numTries} tries left...`);
+      await new Promise((resolve) => setTimeout(resolve, 1_000));
+      numTries--;
+    }
+  }
+}
+
+describe('E2E', () => {
```

Comment:
need to move the other e2e tests out of `e2e/` and into here but will tackle that separately